projects/components/src/filtering/filter-bar/filter-bar.component.scss

```diff
@@ -6,11 +6,13 @@
   flex-direction: column;
   justify-content: center;
   min-height: 36px;
+  overflow-y: scroll;
```

Comment:
This is debatable. Can we get ux team's opinion?

Reply:
Also, this is the advanced filter bar with the same name. It needs refactoring.

Reply:
@anandtiwary intention was to do it on the advance filter bar in the explorer page. the bug was filed for that.